website/docs/src/content/_meta.ts

```diff
@@ -2,12 +2,13 @@ const meta = {
   index: "Overview",
   "basic-concepts": "Basic Concepts",
   quickstart: "Quickstart",
-  cloud: "Cloud (Beta)",
-  "why-jsx": "Why JSX?",
   concepts: "Concepts",
-  examples: "Examples",
+  "why-jsx": "Why JSX?",
```

Comment:
might be worth moving this above concepts - that way all the single files would be above the folders.